casr/src/bin/casr-cluster.rs

```diff
@@ -106,14 +118,25 @@ fn make_clusters(inpath: &Path, outpath: Option<&Path>, jobs: usize) -> Result<u
         bail!("{} valid reports, nothing to cluster...", stacktraces.len());
     }
 
-    let clusters = cluster_stacktraces(&stacktraces)?;
+    // Get clusters
+    let mut clusters = cluster_stacktraces(&stacktraces)?;
 
     // Cluster formation
-    let cluster_cnt = *clusters.iter().max().unwrap();
+    let cluster_cnt: usize = *clusters.iter().max().unwrap();
     for i in 1..=cluster_cnt {
         fs::create_dir_all(format!("{}/cl{}", &outpath.display(), i))?;
     }
+
+    // Get clusters with crashline deduplication
+    if dedup {
+        dedup_crashlines(&crashlines, &mut clusters);
+    }
+
     for i in 0..clusters.len() {
+        // Skip casreps with duplicate crashlines
+        if clusters[i] == 0 {
```

Comment:
Maybe we could count the total duplicates from all clusters? And than return a tuple (clusters, duplicates)?
if duplicates is not 0, we could print some message, that we removed this amount of reports from clusters.

Reply:
Done